integration/plugin/common/plugin_test.go

```diff
@@ -36,3 +50,109 @@ func TestPluginInvalidJSON(t *testing.T) {
 		})
 	}
 }
+
+func TestPluginInstall(t *testing.T) {
+	skip.If(t, testEnv.IsRemoteDaemon, "cannot run daemon when remote daemon")
+	skip.If(t, testEnv.OSType == "windows")
```

Comment:
`skip.If(t, testEnv.IsRootless, "rootless mode has different view of localhost")`